api/src/main/java/org/sefglobal/scholarx/model/BaseScholarxModel.java

```diff
@@ -0,0 +1,33 @@
+package org.sefglobal.scholarx.model;
+
+import java.util.Date;
+
+public class BaseScholarxModel {
```

Comment:
This should be `public abstract class BaseScholarxModel implements Serializable` right? @YohanAvishke